walmart/walmart.py

```diff
@@ -161,6 +164,14 @@ class Items(Resource):
 
     path = 'items'
 
+    def get_items(self):
+        "Get all the items from the Item Report"
+        response = self.connection.report.all(type="item")
+        zf = zipfile.ZipFile(io.BytesIO(response.content), "r")
+        product_report = zf.read(zf.infolist()[0]).decode("utf-8")
+
+        return map(dict, csv.DictReader(io.StringIO(product_report)))
```

Comment:
DictReader returns a list of dict already right?

Reply:
https://docs.python.org/3/library/csv.html#csv.DictReader

in python 3 it returns OrderedDict
See the last line

Reply:
that should still be fine right? The ordereddict keeps the same api as dict?